### ISSUE TITLE
How to get a vue instance？

### ISSUE THREAD

**Issue by Pasoul** [CanopyTax/single-spa-vue#26]
**Describe the bug or question**
We know that vueLifecycles returns a lifecycle object instead of vm. So how do I get vm?
```
// app1.js
const vueLifecycles = singleSpaVue({
  Vue,
  appOptions: {
    el: '#app',
    render: (h) => h(App),
    router,
    store
  }
});
```
```
// Under normal circumstances, I can get Vue instances like this.
const vm = new Vue({
  el: "#app",
  router,
  store,
  render: h => h(App)
});
```
**To Reproduce**

**Expected behavior**
I want to get the vue instance of a child application

**Screenshots and/or console output**

**Additional context**

**Comment by joeldenning** [comment]
I moved this over to the single-spa-vue repo.

It is not currently possible to get the Vue instance, but I think we should add this as a feature of single-spa-vue. See the following line of code:

https://github.com/CanopyTax/single-spa-vue/blob/6dc5ad8b6e7d2d26c0f17da602e59bff7fe9aa82/src/single-spa-vue.js#L86

We aren't exposing that `mountedInstances` variable anywhere though. What do you think about the following proposal?

```js
export const bootstrap = vueLifecycles.bootstrap

// This doesn't work yet, but we could make this work?
export const mount = props => {
  return vueLifecycles.mount(props).then(vueInstance => {
    // do something with the vue instance
  })
}

export const unmount = vueLifecycles.unmount;
```

Would the above suit your needs? Or would it be better to try to get it into the `data` and single-spa custom props, instead?

**Comment by Pasoul** [comment]
Thank you very much, I think this new feature can help me

**Comment by joeldenning** [comment]
@Pasoul the new feature is not yet implemented. Would you be interested in submitting a pull request to add this feature?

**Comment by joeldenning** [comment]
Closed in #28

**Comment by joeldenning** [comment]
https://github.com/CanopyTax/single-spa-vue/releases/tag/v1.6.0
